src/backend/ManagementHub.Storage/Migrations/20251223122500_AddTournamentInvitesAndParticipants.cs

```diff
@@ -0,0 +1,131 @@
+using System;
```

Comment:
Please remove this migration file and revert manual changes to the snapshot in the migrations folder and create it again by calling `dotnet ef migrations add {name}` command from the directory of the Storage project. Make sure to commit changes to the dbContextSnapshot as well as the migration dbContextSnapshot.
You can validate migration was added successfully by calling `dotnet ef migrations list`